firewall/interception/interception.go

```diff
@@ -25,7 +25,18 @@ func Start() error {
 		return nil
 	}
 
-	return start()
+	var ch = Packets
```

Comment:
Would suggest to use a more descriptive name for `ch` everywhere, such as `packetInput` or `packetIngest`.